themes/ohcoe/static/js/scripts.js

```diff
@@ -93,21 +94,30 @@ $(function() {
             }
         });
 
-        // Calulate the percentage of growth for the entire domain
-        var domainScoreGrowth = 0;
-        if (learningObjectiveCounter) {
-            domainScoreGrowth = round(domainScore / learningObjectiveCounter);
-        }
-
-        var scoreContainer = $('#domain-score-' + currentDomain);
-        if (scoreContainer && domainScoreGrowth) {
-            scoreContainer.html(
-                'You have this much growth: <span id="score-container">' +
-                domainScoreGrowth + ' out of 4 </span>');
+        // Domain Progress Bar
+        if (document.getElementById('domain-review')) {
+            var meanPreScore = round(mean(domainPreScore));
```

Comment:
`domainPreScore` can be an empty array here, which causes an error when `mean` is called.

Reply:
added in checks on the arrays being `mean`ed